cli/dcoscli/job/main.py

```diff
@@ -112,12 +112,12 @@ def _cmds():
 
         cmds.Command(
             hierarchy=['job', 'list'],
-            arg_keys=['--json'],
+            arg_keys=['--json', '--quiet'],
             function=_list),
 
         cmds.Command(
             hierarchy=['job', 'history'],
-            arg_keys=['<job-id>', '--json', '--show-failures'],
+            arg_keys=['<job-id>', '--json', '--failures', '--last', '--quiet'],
```

Comment:
changing `--show-failures` to `--failures` might break user scripts, we should avoid BC break unless absolutely necessary.

Reply:
understand... the original flag wasn't thought out.